bucket-version-mismatch/rewrite_timeseries_bucket_version_mismatch.js

```diff
@@ -25,6 +25,11 @@ const coll = db.getCollection(collName);
 // 2) Change the buckets with bucket version mismatch to the correct version.
 // 3) Validate that there are no bucket version mismatches.
 // ----------------------------------------------------------------------------------------
+// We can't pattern match with the entire v2 error message because we include
+// the fieldName of the unsorted v2 bucket.
```

Comment:
Note that we are dependent on these error msgs not changing for validation. Can we depend on this? I believe that we are attempting to make validation compatible for all versions, but wanted to double check (this applies to the `mismatchEmbeddedIdTimestampMsg` variable declared below)